gelataio_boy_control/src/scooping.cpp

```diff
@@ -92,12 +92,17 @@ bool ScoopingMain::scoop_ice(Point start, Point end, std::function<void(bool)> f
     }
 
     if (successful) {
+        this->point_above_cup.x = -0.1;
+        this->point_above_cup.y = -0.4;
+        this->point_above_cup.z = 0.25;
+        ROS_WARN("Before rotating the wrist_right");
         ROS_INFO("Departing from the scoop stage");
-        successful &= this->depart_from_scoop();
+        ROS_WARN("Right now, we were supposed to get this from the Ruofan's CV code");
+        successful &= this->depart_from_scoop(this->point_above_cup);
     }
 
-    ROS_INFO("Going home");
-    successful &= right_arm.goHome();
+    // ROS_INFO("Going home");
+    // successful &= right_arm.goHome();
```

Comment:
Maybe we add a separate service for the going home function. We should definitely go home after the last scoop. The arm should be in home position for driving. If not, we should add this back here. Then it would go home after each of the scoops.

Reply:
+1